openupgrade_scripts/scripts/website_sale_collect/18.0.1.0/post-migration.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2025 Tecnativa - Carlos Lopez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+def _delivery_carrier_map_delivery_type(env):
+    delivery_type_legacy = openupgrade.get_legacy_name("delivery_type")
+    openupgrade.map_values(
+        env.cr,
+        delivery_type_legacy,
+        "delivery_type",
+        [("onsite", "in_store")],
+        table="delivery_carrier",
+    )
+
+
+def _payment_provider_map_custom_mode(env):
+    custom_mode_legacy = openupgrade.get_legacy_name("custom_mode")
+    openupgrade.map_values(
+        env.cr,
+        custom_mode_legacy,
+        "custom_mode",
+        [("onsite", "on_site")],
+        table="payment_provider",
+    )
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.m2o_to_x2m(
+        env.cr,
+        env["delivery.carrier"],
+        "delivery_carrier",
+        "warehouse_ids",
+        "warehouse_id",
+    )
+    _delivery_carrier_map_delivery_type(env)
+    _payment_provider_map_custom_mode(env)
+    openupgrade.delete_record_translations(
```

Comment:
Why this?

Reply:
Yes, translations, but then it means there are different values that you need to fill, so yes, noupdate_changes.xml.